src/Glpi/Asset/Capacity/HasAntivirusCapacity.php

```diff
@@ -51,6 +52,12 @@ public function getIcon(): string
         return ItemAntivirus::getIcon();
     }
 
+    #[Override]
+    public function getDescription(): string
+    {
+        return __("List antivirus software(s) found by automatic inventory");
```

Comment:
This is not only from inventory
```suggestion
        return __("List antivirus software(s)");
```